### PR TITLE
Update why-learn-csharp.md

### PR DESCRIPTION
**Pull request by okgamr**

### CODE REVIEW

**Comment by enki-hq** [comment]
Failed parsing files: 
  - csharp/csharp-core/intro-to-csharp/why-learn-csharp.md: 
 > Cannot compile node with invalid type questionHeadline

**Comment by okgamr** [comment]
> Failed parsing files:
> 
> * csharp/csharp-core/intro-to-csharp/why-learn-csharp.md:
> 
> > Cannot compile node with invalid type questionHeadline

please explain, @enki-hq

**Comment by nem035** [comment]
Hi @okgamr, thanks for your contribution ❤️ 

The error you're seeing is because you change the main heading to be an h3: 

```
### Why It's Good to Learn C-Sharp
 ^^ you added these two #'s
```

This is not allowed in Enki insights as the main heading is the title of the insight, and it's required.

Regarding your other suggestions, they seem to be stylistic changes that don't necessarily improve anything about the lesson itself and are more so focused on subjective visual changes. 

Can you elaborate a bit on why you're suggesting them? What is the improvement you're trying to make?

**Comment by okgamr** [comment]
Well, it looks more organized by that. Getting the big heading (aka title) and the points after it, but smaller so it distinguishes. If I see the one '''#''' looks well, maybe I won't just give up 😃

**Comment by enki-hq** [comment]
Failed parsing files: 
  - csharp/csharp-core/intro-to-csharp/why-learn-csharp.md: 
 > Cannot compile section node with name -- # Content

**Comment by okgamr** [comment]
Now sau what's wrong @enki-hq

**Comment by enki-hq** [comment]
Failed parsing files: 
  - csharp/csharp-core/intro-to-csharp/why-learn-csharp.md: 
 > Cannot compile node with invalid type thematicBreak

**Comment by okgamr** [comment]
Review again please

**Comment by okgamr** [comment]
> Cannot compile node with invalid type thematicBreak

What is `thematicBreak`?

**Comment by okgamr** [comment]
@enki-hq

File: csharp/csharp-core/intro-to-csharp/why-learn-csharp.md [hunk 1]
```diff
@@ -11,20 +11,20 @@ category: must-know
 
 ---
 
-## Content
+# Content
```
Comment (nem035) [inline csharp/csharp-core/intro-to-csharp/why-learn-csharp.md:14]:
```suggestion
## Content
```

We only allow one H1 per insight, the content section must be H2 like before

Comment (nem035) [reply csharp/csharp-core/intro-to-csharp/why-learn-csharp.md:14]: BTW You can read more about the format here: https://github.com/enkidevs/curriculum/wiki/Insight-Documentation

Comment (okgamr) [reply csharp/csharp-core/intro-to-csharp/why-learn-csharp.md:14]: Yay, I'll see and rewrite ig

**Comment by okgamr** [comment]
just that thing. JUST THAT
now we got a better mental map

**Comment by okgamr** [comment]
@nem035 can you make the review now?

**Comment by Stefan-Stojanovic** [comment]
Thank you for the contribution, @okgamr 🎉

**Comment by okgamr** [comment]
🎉🥳

**Pull request merged**